cookbooks/bcpc/templates/default/zabbix/server.conf.erb

```diff
@@ -5,7 +5,7 @@
 # Local changes will be reverted.
 #
 
-SourceIP=<%="#{node[:bcpc][:management][:vip]}"%>
+SourceIP=<%="#{node[:ipaddress]}"%>
```

Comment:
According to https://docs.chef.io/ohai.html#automatic-attributes, "if the node has a default route, this is the IPV4 address for the interface.". Since we have bond0 as the default route, node[:ipaddress] will always use the ip of it